google/datalab/contrib/pipeline/_pipeline.py

```diff
@@ -30,13 +30,19 @@ class Pipeline(object):
 from airflow.contrib.operators.bigquery_to_bigquery import BigQueryToBigQueryOperator
 from airflow.contrib.operators.bigquery_to_gcs import BigQueryToCloudStorageOperator
 from airflow.contrib.operators.gcs_to_bq import GoogleCloudStorageToBigQueryOperator
-from google.datalab.contrib.bigquery.operators.bq_load_operator import LoadOperator
-from google.datalab.contrib.bigquery.operators.bq_execute_operator import ExecuteOperator
-from google.datalab.contrib.bigquery.operators.bq_extract_operator import ExtractOperator
+from google.datalab.contrib.bigquery.operators._bq_load_operator import LoadOperator
+from google.datalab.contrib.bigquery.operators._bq_execute_operator import ExecuteOperator
+from google.datalab.contrib.bigquery.operators._bq_extract_operator import ExtractOperator
 from datetime import timedelta
 from pytz import timezone
 """
 
+  # These are documented here: https://airflow.incubator.apache.org/code.html?highlight=macros#default-variables  # noqa
+  _airflow_macros = {
+    'ds': '{{ ds }}',  # the execution date as YYYY-MM-DD
+    'ts': '{{ ts }}',  # same as execution_date.isoformat()
```

Comment:
Can you just put an example, without referring to execution_date which isn't one of the listed macros?

Reply:
Understood the confusion. Have added more descriptive comments with the full format; hopefully this will address it. Also, I've diverged from airflow's macros by internally mapping 'ds' and 'ts' to 'ds_nodash' and 'ts_nodash'. Since users are likely to use this in file names, I felt that this was safer to do. Additionally, in the spirit of providing an abstraction that was layered on top / simpler than Airflow, I didn't feel compelled to match Airflow's macros 1:1.